pokemon-checker/src/DataTransferObjects/MoveDTO.ts

```diff
@@ -1,31 +1,35 @@
 
-enum moveType {
+export enum damageClass {
   PHYSICAL = "physical",
   SPECIAL = "special",
-  STATUS = "status"
+  STATUS = "status",
+  NULL = "null"
```

Comment:
I've added a `null` type to support move stubs created by the factory.

Reply:
That's good, there might actually be a NULL type for moves (like Follow Me or Baton-pass or something)
I think a pass over the Move List in the API might help